apps/editing-toolkit/.gitignore

```diff
@@ -3,3 +3,4 @@
 /editing-toolkit-plugin/*/dist/
 /editing-toolkit-plugin/newspack-blocks/synced-newspack-blocks
 /.wp-env.override.json
+/.cache/
```

Comment:
do we need a similar entry for the other apps? If so, maybe it would be better to have a wildcard in the root .gitignore?

Reply:
I still don't really know where it came from so not sure. My assumption is it's part of our common webpack config, in which case yeah it should probably just have a wildcard in the root ignore file.

I guess really how likely is it that someone would want to commit files in a hidden folder called `.cache`. tbh it probably won't hurt anyone if I ignore `/apps/*/.cache/`

Reply:
Moved the ignore pattern to the root `.gitignore`